### PR TITLE
Sutton spr23

### PR DESCRIPTION
**Pull request by JonahTS**
Changed Cadre Info/Pictures , changed join date to Spring 2025. Waiting to receive updated NSE packets but cadre wants to go ahead and get the pictures changed.

### CODE REVIEW

File: content/contact.hbs [hunk 1]
```diff
@@ -16,7 +16,11 @@
 		<div class="container">
 				<div class="columns">
 					<div class="column is-full">
-						<h1>Join in <span class="semester-text">Spring 2024</span> or later</h1>
+<<<<<<< HEAD
```
Comment (petschekr) [inline content/contact.hbs]: I think you have a leftover merge conflict here

File: content/join.hbs [hunk 2]
```diff
@@ -25,17 +25,17 @@
 			</ol>
 			<p class="has-text-centered">This list is not exhaustive; other entrance standards may apply.</p>
 
-			<h1>Join in the <span class="semester-text">Fall 2023</span> semester</h1>
+			<h1>Join in the <span class="semester-text">Spring 2025</span> semester</h1>
 			<div class="columns">
 				<div class="column is-half side-by-side current-semester">
 					<h2>Join as a <span class="semester-text">Georgia Tech</span> student</h1>
-					<p>To enroll for Fall 2023, please first review the <a href="files/Enrollment Information Packet Fall 2023.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
+					<p>To enroll for Spring 2025, please first review the <a href="files/Enrollment Information Packet_Spring 2023.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
```
Comment (petschekr) [inline content/join.hbs]: Make sure the file URL matches what's in `rendered/files`. I think this should be `files/Enrollment Information Packet Spring 2024.pdf`

File: content/join.hbs [hunk 3]
```diff
@@ -25,17 +25,17 @@
...
 					<a class="button is-large" href="files/GT NSE packet.pdf">NSE Packet for Georgia Tech Students</a>
 				</div>
 				<div class="column is-half side-by-side future-semester">
 					<h2>Join as a <span class="semester-text">crosstown</span> student</h1>
 					<p class="college-list">Detachment 165 hosts students from:<br />Agnes Scott College, Clark Atlanta University, Clayton State University, Emory University, Georgia State University, Kennesaw State University, Morehouse College, Oglethorpe University, Spelman College, Mercer University, and University of West Georgia</p>
-					<p>To enroll for Fall 2023, please first review the <a href="files/Enrollment Information Packet Fall 2023.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>
+					<p>To enroll for Spring 2025, please first review the <a href="files/Enrollment Information Packet_Spring 2023.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>
```
Comment (petschekr) [inline content/join.hbs]: Same here

File: content/join.hbs [hunk 4]
```diff
@@ -91,7 +91,11 @@
 			<p class="has-text-centered">To apply to AFROTC and complete the application checklist, follow this guide.</p>
 			<p class="has-text-centered"><a class="button is-large future-interest" href="files/WINGS Application Guide.pdf">WINGS Application Guide</a></p>
 
-			<h1>Join in <span class="semester-text">Spring 2024</span> or later</h1>
+<<<<<<< HEAD
```
Comment (petschekr) [inline content/join.hbs]: Another merge conflict that got left over. I think you want to keep the `<h1>Join in <span class="semester-text">Spring 2025</span> or later</h1>` and delete the rest

**Review by petschekr — CHANGES_REQUESTED** [review]
Super minor stuff

**Comment by JonahTS** [comment]
Okay, I pushed it again, I think I fixed the merge conflicts

File: content/join.hbs [hunk 5]
```diff
@@ -25,17 +25,17 @@
 			</ol>
 			<p class="has-text-centered">This list is not exhaustive; other entrance standards may apply.</p>
 
-			<h1>Join in the <span class="semester-text">Fall 2023</span> semester</h1>
+			<h1>Join in the <span class="semester-text">Spring 2025</span> semester</h1>
```
Comment (petschekr) [inline content/join.hbs:28]: I think this should be Fall 2024 since it's for the upcoming semester

File: content/join.hbs [hunk 6]
```diff
@@ -25,17 +25,17 @@
 			</ol>
 			<p class="has-text-centered">This list is not exhaustive; other entrance standards may apply.</p>
 
-			<h1>Join in the <span class="semester-text">Fall 2023</span> semester</h1>
+			<h1>Join in the <span class="semester-text">Spring 2025</span> semester</h1>
 			<div class="columns">
 				<div class="column is-half side-by-side current-semester">
 					<h2>Join as a <span class="semester-text">Georgia Tech</span> student</h1>
-					<p>To enroll for Fall 2023, please first review the <a href="files/Enrollment Information Packet Fall 2023.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
+					<p>To enroll for Spring 2025, please first review the <a href="files/Enrollment Information Packet Spring 2024.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
```
Comment (petschekr) [inline content/join.hbs:32]: Fall 2024

File: content/join.hbs [hunk 7]
```diff
@@ -25,17 +25,17 @@
...
 					<a class="button is-large" href="files/GT NSE packet.pdf">NSE Packet for Georgia Tech Students</a>
 				</div>
 				<div class="column is-half side-by-side future-semester">
 					<h2>Join as a <span class="semester-text">crosstown</span> student</h1>
 					<p class="college-list">Detachment 165 hosts students from:<br />Agnes Scott College, Clark Atlanta University, Clayton State University, Emory University, Georgia State University, Kennesaw State University, Morehouse College, Oglethorpe University, Spelman College, Mercer University, and University of West Georgia</p>
-					<p>To enroll for Fall 2023, please first review the <a href="files/Enrollment Information Packet Fall 2023.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>
+					<p>To enroll for Spring 2025, please first review the <a href="files/Enrollment Information Packet Spring 2024.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>
```
Comment (petschekr) [inline content/join.hbs:38]: Fall 2024

File: content/join.hbs [hunk 8]
```diff
@@ -91,7 +91,8 @@
 			<p class="has-text-centered">To apply to AFROTC and complete the application checklist, follow this guide.</p>
 			<p class="has-text-centered"><a class="button is-large future-interest" href="files/WINGS Application Guide.pdf">WINGS Application Guide</a></p>
 
-			<h1>Join in <span class="semester-text">Spring 2024</span> or later</h1>
+			<h1>Join in <span class="semester-text">Spring 2025</span> or later</h1>
+			<h1>Join in <span class="semester-text">Fall 2024</span> or later</h1>
```
Comment (petschekr) [inline content/join.hbs:95]: Got a duplicate line here. I think this should be Spring 2025 since it's for future interest.

**Review by petschekr — CHANGES_REQUESTED** [review]

**Comment by petschekr** [comment]
Superseded by #72

**Pull request closed without merging**